pluto/src/main/java/com/mocklets/pluto/Pluto.kt

```diff
@@ -57,4 +59,8 @@ object Pluto {
     fun setANRListener(listener: ANRListener) {
         anrSupervisor.setListener(listener)
     }
+
+    fun setCustomActions(items: List<CustomAction>) {
```

Comment:
this will override previously set actions, please refer to `setAppProperties()` method implementation.